platform/ui/src/components/InputSelect/InputSelect.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Select, InputLabelWrapper } from '@ohif/ui';
+
+const InputSelect = ({
+  label,
+  isSortable,
+  sortDirection,
+  onLabelClick,
+  value,
+  placeholder,
+  options,
+  onChange,
+}) => {
+  return (
+    <InputLabelWrapper
+      label={label}
+      isSortable={isSortable}
+      sortDirection={sortDirection}
+      onLabelClick={onLabelClick}
+    >
+      <Select
+        placeholder={placeholder}
+        className="mt-2"
+        options={options}
+        value={value}
+        isMulti={true}
```

Comment:
`<Select />` will always be `multi`?

Reply:
I'm okay with it being locked down and potentially renaming this `MultiSelect`

Again, less options is better, unless we need that level of configuration. It's less to test, and these are UI states we won't be able to verify the accuracy of as they're not encountered in the application.

If we find ourselves needing a non-multi-select, then we can revisit the API here?

Reply:
Perfect! I agree. Let's keep as it is for now!

Reply:
I thoguht the same as Danny, as he is always suggesting to lower the options if we can, as our input field is a multiSelect for now, I decided to move with it locking this option. 

But I definetely liked the idea of renaming to InputMultiSelect.